shopping-cart.html

```diff
@@ -104,6 +104,98 @@ <h2 class="shopping-cart__left-text">Mi carrito</h2>
+                      <path d="M13.0046 7.33333V11.3333" stroke="#A6A6A6" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                    </svg>Eliminar </a>
+                  <div class="purchase-summary__quantity-selector">
+                    <button class="purchase-summary__decrease"></button>
+                    <span class="purchase-summary__quantity">1</span>
+                    <button class="purchase-summary__increase"></button>
+                    <button class="purchase-summary__increase purchase-summary__increase--1"></button>
+                  </div>
+                </div>
+              </div>
+              <div class="purchase-summary__analytics-bord purchase-summary__analytics--bord-2"></div>
+            </div>
+          </div>
+        </div>
+        <div class="purchase-summary__payment">
+          <a href="#" class="purchase-summary__analytics__link"></a>
+          <div class="purchase-summary__payment-row">
+            <span class="purchase-summary__payment-code">Subtotal</span>
+            <h4 class="purchase-summary__payment-precio">$359,98</h4>
+          </div>
+          <div class="purchase-summary__payment-row">
+            <span class="purchase-summary__payment-code">IBTMS</span>
+            <h4 class="purchase-summary__payment-precio">$20.99</h4>
+          </div>
+          <div class="purchase-summary__payment-total">
+            <span class="purchase-summary__payment-quantity">Total</span>
+            <h1 class="purchase-summary__payment-precio purchase-summary__payment-precio--3">$380,97</h1>
+          </div>
+          <div class="purchase-summary__payment-shipping">
+            <img class="purchare-summary__payment-truck" src="image/Truck.png" alt="">
+            <p class="purchase-summary__payment-available">Disponible para envío gratis</p>
+          </div>
+          <a class="purchase-summary__payment-process">
+            <p class="purchase-summary__payment-text">PROCESAR EL PAGO </p>
+            <svg width="17" height="17" viewBox="0 0 17 17" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path fill-rule="evenodd" clip-rule="evenodd" d="M2.91589 9.34584C2.91589 8.24127 3.81132 7.34583 4.91589 7.34583H12.9159C14.0205 7.34583 14.9159 8.24127 14.9159 9.34583V12.6792C14.9159 13.7837 14.0205 14.6792 12.9159 14.6792H4.91589C3.81132 14.6792 2.91589 13.7837 2.91589 12.6792V9.34584Z" stroke="white" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+              <path d="M5.58252 7.34583V4.67917C5.58252 2.83822 7.0749 1.34583 8.91585 1.34583C10.7568 1.34583 12.2492 2.83822 12.2492 4.67917V7.34583" stroke="white" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+            </svg>
+          </a>
+          <div class="purchase-summary__payment-methods">
+            <img class="purchase-summary__payment-img" src="image/payment-1.jpg" alt="punto pago">
```

Comment:
Quite la clase y no vi cambio alguno en el navegador
```suggestion
            <img src="image/payment-1.jpg" alt="punto pago">
```